ArduPlane/quadplane.cpp

```diff
@@ -1541,6 +1547,14 @@ void SLT_Transition::update()
             // will stop stabilizing
             throttle_scaled = 0.01;
         }
+        if (quadplane.tiltrotor.enabled() && !quadplane.tiltrotor.has_vtol_motor() && !quadplane.tiltrotor.has_fw_motor()) {
```

Comment:
has this been tested on a tilt tri? which would appear to be the same as an all tilting, but itsnt?